stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -523,10 +521,12 @@ private long applyOperation(String filePath) throws IOException {
     }
 
     private void finishProcessingFiles() {
-      if (FuseIOOperation.isRead(mParameters.mOperation)) {
-        throw new IllegalArgumentException(String.format("Thread %d finishes reading all its files "
-            + "before the bench ends. For more accurate result, use more files, or larger files, "
-            + "or a shorter duration", mThreadId));
+      if (FuseIOOperation.isRead(mParameters.mOperation)
+          || (mParameters.mOperation == FuseIOOperation.WRITE
```

Comment:
This reminds me we actually set duration to 0 if the operation is `Write`...Otherwise looks good

Reply:
nvm I'm blind